src/mds/Server.cc

```diff
@@ -508,7 +508,7 @@ void Server::handle_client_reclaim(const cref_t<MClientReclaim> &m)
     return;
   }
 
-  std::string_view fs_name = mds->get_fs_name();
+  std::string_view fs_name = mds->mdsmap->get_fs_name();
```

Comment:
Patrick, I will add an another PR as you suggested that removes the fs_name from MDSRank;  set map_fs_name in the MMDSMap message to null along with the comment that this was accidentally added when we thought we needed to pass fsid in the messages.